src/components/KTable/KTable.vue

```diff
@@ -207,6 +207,8 @@ export interface TableHeader {
   label: string
   sortable: boolean
   hideLabel: boolean
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  useSortHandlerFn: Function
```

Comment:
TBH this isn't a valid type 

Could do something like

`useSortHandlerFn: (payload: Record<string, any>) => void`

If you wanted to get real fancy, you could add a type interface for payload instead of using this default object.

Reply:
Fixed